src/olympia/editors/forms.py

```diff
@@ -231,6 +231,110 @@ def filter_qs(self, qs):
         return qs
 
 
+class AllAddonSearchForm(happyforms.Form):
+    text_query = forms.CharField(
+        required=False,
+        label=_lazy(u'Search by add-on name / author email / guid'))
+    searching = forms.BooleanField(widget=forms.HiddenInput, required=False,
+                                   initial=True)
+    admin_review = forms.ChoiceField(required=False,
+                                     choices=[('', ''),
+                                              ('1', _lazy(u'yes')),
+                                              ('0', _lazy(u'no'))],
+                                     label=_lazy(u'Admin Flag'))
```

Comment:
The indentation here and for `searching` and `deleted` doesn't match the other fields, can we make this the same? (I like how `text_query`, `application_id` and others are indented more.